frontend/src/models/index.tsx

```diff
@@ -149,6 +149,7 @@ export type Settings = {
     time: string;
   };
   updatedAt?: Date;
+  navbarTitle?: string;
```

Comment:
Can you also add a default value for this property in the `context/SettingsProvider.ts` please? — Set the default value to 'Performance Dashboard' just like you did in the backend.